src/main/java/org/c4sg/controller/ProjectController.java

```diff
@@ -222,19 +222,19 @@ public String retrieveImage(@ApiParam(value = "Project id to get image for", req
     @ApiOperation(value = "Deletes a project's images")
     public String deleteImage(@ApiParam(value = "Project id to delete image for", required = true)
     							@PathVariable("id") int id) {
-    	ProjectDTO p = projectService.findById(id);
+    	ProjectDTO pr = projectService.findById(id);
     	File image = new File(projectService.getImageUploadPath(id));
     	try {
     		boolean del = image.delete();
-    		p.setImageUrl(null);
-    		projectService.updateProject(p);
+    		pr.setImageUrl(null);
+    		projectService.updateProject(pr);
     		if (del) {
     			return "Success";
```

Comment:
Use `ResponseEntity.noContent().build();` in the return statement. You can take the method `getApplicants` in the `UserController` class as an example to help you.